README.md

```diff
@@ -63,5 +71,5 @@ end
 Inky.set_pixels(InkySample, painter, border: :white)
 
 # Flip a few pixels
-Inky.set_pixels(pid, %{{0,0}: :black, {3,49}: :red, {23, 4}: white})
+Inky.set_pixels(InkySample, %{{0, 0} => :black, {3, 49} => :red, {23, 4} => :white})
```

Comment:
Why the change here?

Reply:
Does it work on your machine? I thought it was syntax error.

Reply:
My bad, it's been a while since I sat down properly with elixir, but I'm pretty sure this worked at some point 🤔...

`%{a: 1, b: 2}`, for example is just fine. Anyway, the example should work, so this is a great change, thanks! I'd just leave the pid in the call to `set_pixels`. Looks good apart from that :)